staging/src/k8s.io/apiserver/pkg/admission/plugin/resourcequota/controller.go

```diff
@@ -434,7 +434,7 @@ func CheckRequest(quotas []corev1.ResourceQuota, a admission.Attributes, evaluat
 		}
 		limitedResourceNames = limitedByDefault(deltaUsage, limitedResources)
 	}
-	limitedResourceNamesSet := quota.ToSet(limitedResourceNames)
+	limitedResourceNamesSet := sets.Set[string](quota.ToSet(limitedResourceNames))
```

Comment:
Casting to `sets.Set` here to avoid touching `quota.ToSet`, otherwise this refactor will get out of hand @cici37.